esp/esp/program/controllers/lottery.py

```diff
@@ -68,6 +69,16 @@ def __init__(self, program, **kwargs):
         
 
         self.program = program
+
+        if Tag.getProgramTag('program_size_by_grade', self.program):
+            # TODO(benkraft): Consider implementing this.  Stanford's use case
+            # as of Fall 2015 (for which program_size_by_grade was written)
+            # doesn't need it, but we might want to implement it anyway, or
+            # remove the program size logic from the lottery entirely.
+            print ("WARNING: The lottery doesn't support the "
+                   "program_size_by_grade Tag yet.  It will run based on "
+                   "`Program.program_size_max`, which is %s." %
+                   self.program.program_size_max)
```

Comment:
Fair point, will fix.  I don't think it's worth printing a bunch of information; the user should either realize "this is an issue, let me fix it" and investigate themselves, or realize "I know, it's fine" and continue.  Prompting the user is something the lottery script doesn't currently do and I'd like to keep it that way without a stronger reason to the contrary.

And yes, we're not handling OverrideFull correctly.  I'm not going to fix that now.  My inclination is that the eventual approach should be to remove program caps from here, because I think letting students mark classes in the lottery and *then* picking which ones get into the program is a poor approach that we need not support (in addition to being one that no chapters currently use).